README.md

```diff
@@ -62,6 +62,8 @@ pipx install uv
 brew install uv
 ```
 
+See the [advanced installation](#advanced-installation) documentation for more details.
```

Comment:
Can we remove this?

Reply:
I really do not want people to have to scroll to the bottom of the documentation to discover how to pin the version with the standalone installer. Can we compromise here? I'm happy to rephrase it.

Reply:
But this doesn't actually tell people how to pin versions with the standalone installer. Is it more useful than having to Command + F for "install"? (If it's that important, should it be up here in the preamble?)

Reply:
E.g., could the whole change just be a line like:

```
# Or a specific version.
curl -LsSf https://astral.sh/uv/0.2.6/install.sh | sh
```

In the above code block? Isn't that more economical than adding this to the real estate of the Getting Started section?

Reply:
> But this doesn't actually tell people how to pin versions with the standalone installer.

That's what I was getting at when I said I'd rephrase.

>  Isn't that more economical than adding this to the real estate of the Getting Started section?

Yeah. So that was my first thought, but there are two small problems

1. I don't want the getting started to suggest installing an out of date version. We can resolve this by adding the README to rooster's target files though so it's always the latest in the example.
2. It's a little awkward given that we split between unix and windows, `# Or, a specific version on MacOS and Linux`?

I'd prefer this honestly just not sure how to phrase it since it doesn't quite fit into the style of the rest of the code block.

Reply:
Okay: https://github.com/astral-sh/uv/pull/4105